draft-ietf-scitt-architecture.md

```diff
@@ -683,15 +683,12 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 # Privacy Considerations
 
-Transparency Services are often publicly accessible.
-Issuers should treat Signed Statements (rendering them as Transparent Statements) as publicly accessible.
-In particular, a Signed Statement Envelope and Statement payload should not carry any private information in plaintext.
+Transparency Services MAY support anonymous access, Issuers MUST ensure Signed Statements submitted to public access services are acceptable for public disclosure.
+In this case, Signed Statements MUST NOT carry confidential information.
+Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
+In some deployments, a Relaying Party, such as an Auditor, might require access to Signed Statements and Statements, which can be made available through adjacent services.
 
-Transparency Services can have an authorization policy controlling who can access the Append-only Log.
-While this can be used to limit who can read the Log, it may also limit the usefulness of the system.
-
-Some jurisdictions have a Right to be Forgotten.
-However, once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
+Transparency Services MUST enforce access control to the Append-only Log.
```

Comment:
In _most_ scenarios, I'd agree writes should be gated, but there are scenarios where all events would be written. For reads, most scenarios likely don't require access control, other than a means to apply throttling.

Reply:
Access control does not mean you cannot have anonymous read or write, just that you have means to control it. I have worked on many systems that a have notion of anonymous read or read for unauthenticated access scenarios. Does that mean those systems, writ large, don't have access control?

There also appears to be informative and useful context in the edits about the last requirement on line 692. Is your criticism about this requirement specifically or all the edits as a whole?

Reply:
@aj-stein-nist, I agree access control does not mean it can't also support anonymity. However, as written with "MUST", it does imply this, which is what I'm basic my objection to this line, as written. However, you are also correct that I had concerns with the generality of the additions as we've worked hard to reduce the ledger to recording cryptographic hashes of the content (signed statements, statements/aka payload/aka content) directly on the ledger.